plugins/experimental/stale_while_revalidate/stale_while_revalidate.c

```diff
@@ -526,11 +560,14 @@ main_plugin(TSCont cont, TSEvent event, void *edata)
 
           TSDebug(PLUGIN_NAME, "set state as async");
           state->async_req = true;
+          // cannot free with main Txn --> disable TXN_CLOSE event
+          TSHttpTxnArgSet(txn, state->plugin_config->txn_slot, NULL);
+          state->resp_info = NULL; // non-blocking internal TSHttpConnect()
           TSHttpTxnCacheLookupStatusSet(txn, TS_CACHE_LOOKUP_HIT_FRESH);
           // TSHttpTxnReenable(txn, TS_EVENT_HTTP_CONTINUE);
           fetch_cont = TSContCreate(fetch_resource, TSMutexCreate());
           TSContDataSet(fetch_cont, (void *)state);
-          TSContSchedule(fetch_cont, 0, TS_THREAD_POOL_TASK);
+          TSContSchedule(fetch_cont, 0, TS_THREAD_POOL_NET);
```

Comment:
Apparently also needed in later versions.   TASK threads cannot start a connection using TSHttpConnect() ... that I can see.